maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -91,6 +95,9 @@
 	@Parameter(defaultValue = "${session}", readonly = true)
 	private MavenSession				session;
 
+	@Parameter
+	private Set<Scope>					scopes	= EnumSet.of(Scope.compile, Scope.runtime, Scope.test);
```

Comment:
Isn't adding `test` a change from before when it was only compile and runtime? Does this present any backwards compatibility issues?

Reply:
What about this question?

Reply:
I suppose I could unset `test` from the defaults here.

Reply:
fixed!